sbndaq-artdaq/Generators/Common/CAENV1730Readout_generator.cc

```diff
@@ -1267,6 +1319,7 @@ void sbndaq::CAENV1730Readout::stop()
     TLOG_INFO("CAENV1730Readout") << "stop()" << TLOG_ENDL;
   TLOG_ARB(TSTOP,TRACE_NAME) << "stop()" << TLOG_ENDL;
 
+  if( (fOutputClk == 1) | (fOutputClkPhase == 1) ){ StopClkToTrgOut(); } 
```

Comment:
Is `StopClkToTrgOut()` really needed?

At ICARUS in the past we used an external script to put the clocks in output (basically the same thing you are doing here, just outside of the main DAQ) and for that we usually just ran `CAEN_DGTZ_Reset()` to restore everything back to normal. Would you be able to check if that works here as well?

If that's the case, since the reset is already in  `configure()`, it is really not needed here in `stop()` either and I would remove this call. Once you stop and terminate, it will reset when it configures again.

Reply:
CAEN_DGTZ_Reset() does run at the beginning of every daq run so that should do the trick. Lemme do a sanity daq run check and propagate this change.

Reply:
Did a sanity check. Is StopClkToTrgOut() is not needed. Will edit code to address this.